model/src/main/java/net/spals/oembed4j/model/OEmbedResponse.java

```diff
@@ -25,6 +25,11 @@
 @JacksonXmlRootElement(localName = "oembed")
 public interface OEmbedResponse {
 
+    /**
+     * As per the oEmbed spec, version should always be 1.0
+     */
+    String DEFAULT_VERSION = "1.0";
```

Comment:
I've read its best practice is not to place constants in interfaces.

http://stackoverflow.com/questions/320588/interfaces-with-static-fields-in-java-for-sharing-constants